core/roboconf-messaging-rabbitmq/src/main/java/net/roboconf/messaging/rabbitmq/internal/RabbitMqClient.java

```diff
@@ -71,13 +72,21 @@
 	 */
 	static final String DEFAULT_PASSWORD = "guest";
 
+
 	protected final Logger logger = Logger.getLogger(this.getClass().getName());
 	protected final String messageServerIp, messageServerUsername, messageServerPassword;
+	private final WeakReference<ReconfigurableClient<?>> reconfigurable;
 	protected LinkedBlockingQueue<Message> messageQueue;
 	Channel channel;
-	private final WeakReference<ReconfigurableClient<?>> reconfigurable;
 
 
+	/**
+	 * Constructor.
```

Comment:
Actually, I like Javadoc comments to separate methods.
With syntax highlighting, it makes the code look better. Just code is so ugly...